### ISSUE TITLE
Multiplayer Bomber example crashes

### ISSUE THREAD

**Issue by Vine2D** [godotengine/godot#38549]
v3.2.1.stable.official

This error occurred on a remote client after all clients were spamming bombs;
`ERROR: RPC 'exploded' is not allowed on node /root/world/players/805868251 from: 1. Mode is 2, master is 805868251.
   At: core/io/multiplayer_api.cpp:291`

**Comment by Calinou** [comment]
How many clients were connected on the server?

**Comment by Vine2D** [comment]
Godot version: v3.2.1.stable.official

OS/device including version: Windows 10

Issue description: Multiplayer Bomber example crashes when spamming bombs in different positions with 3 clients connected. 2 clients were remote.
`ERROR: RPC 'exploded' is not allowed on node /root/world/players/805868251 from: 1. Mode is 2, master is 805868251. At: core/io/multiplayer_api.cpp:291`

Steps to reproduce: Have all players repeatedly drop bombs near each other. May be involved with other players receiving damage or the game ending.

Minimal reproduction project: n/a

**Comment by KoBeWi** [comment]
Isn't this demo issue?

**Comment by Faless** [comment]
Most likely fixed in recent Godot 3 and 4 after #38119 . Closing as there have been no recent reports of such crashes, but feel free to comment (or open a new issue) if you are still experiencing it.
